MyToolsYourToolsBackend/MyToolsYourToolsBackend.Application/Services/OfferService.cs

```diff
@@ -58,6 +58,17 @@ public bool CheckIfOfferIsActive(Guid id)
             return _dbContext.Offers.FirstOrDefault(o => o.Id == id).Status == OfferStatus.Active;
         }
 
+        public void DeleteOffer(Guid id)
+        {
+            var offerToDelete = _dbContext.Offers.FirstOrDefault(o => o.Id == id);
+            _dbContext.Remove(offerToDelete);
```

Comment:
Ej a jak usunie się ofertę to wszystkie wypozyczenia i powiadomienia z nią związane są usuwane kaskadowo? bo jak nie trzebaby to ręcznie zrobić, bo się może później wykrzaczyć program